qemu/tests/cfg/block_iscsi_format_large_size_disk.cfg

```diff
@@ -0,0 +1,43 @@
+- block_iscsi_format_large_size_disk:
+    virt_test_type = qemu
+    type = block_iscsi_format_large_size_disk
+    only Windows
+    start_vm = no
+    data_images="stg1"
+    images += " ${data_images}"
+    drive_werror = stop
+    drive_rerror = stop
+    force_create_image_stg1 = yes
+    force_remove_image_stg1 = yes
+    block_raw_name = large-disk
+    image_name_stg1 = images/${block_raw_name}
+    image_format_stg1 = raw
+    image_size_stg1 = 100G
+    target = iqn.2019-12.com.redhat:kvm-qe
+    emulated_image = ${image_name_stg1}.raw
+    emulated_image_size = ${image_size_stg1}
+    clean_cmd="dd if=/dev/zero of=%s count=256 bs=1M oflag=direct && sync && partprobe"
+    cmd_set_max_sector = "echo 64 > /sys/block/{}/queue/max_sectors_kb"
+    cmd_get_max_segment = cat /sys/block/{}/queue/max_segments
+    file_size = 2147483648
+    guest_cmd = "WIN_UTILS:\coreutils\DummyCMD.exe {} ${file_size} 1"
+    Windows:
+      labeltype = msdos
+      fstype = ntfs
+    Linux:
```

Comment:
This seems meaningless if above limit   ‘only Windows ‘ on line4

Reply:
updated.